org.lflang/src/org/lflang/generator/TypeScriptGenerator.xtend

```diff
@@ -335,7 +335,6 @@ class TypeScriptGenerator extends GeneratorBase {
 
         // If this is a federated execution, generate C code for the RTI.
         if (isFederated) {
-            createFederateRTI()
```

Comment:
Why is the TypeScript generator copying all these C files?  Maybe this was there only for the RTI?  No longer relevant?

Reply:
Ah, great catch! This piece of code seemed to only be relevant for the RTI.

Reply:
Yes, Hokeun and I have plans to completely clean up `TypeScriptGenerator`. After this pull request has been merged in we'll be able to get rid of RTI-related code in that class.